### PR TITLE
Allow extra args for non-backup commands

### PR DESCRIPTION
**Pull request by nettnikl**
Add the env variable `RESTIC_EXTRA_ARGS` that will be added to _all_ `restic` commands, including `restic stats` etc. This can be used for additional setting for accessing the repo (like the sftp binary setting).

### CODE REVIEW

**Comment by nettnikl** [comment]
You mentioned it's probably too specific, but i'd like to drop this mr anyways, it's quite lightweight.

File: bin/restic_backup.sh [hunk 1]
```diff
@@ -108,7 +114,7 @@ restic backup \
 	--tag "$RESTIC_BACKUP_TAG" \
 	"${B2_ARG[@]}" \
 	"${exclusion_args[@]}" \
-	"${extra_args[@]}" \
```
Comment (erikw) [inline bin/restic_backup.sh:111]: As my understanding is that `RESTIC_EXTRA_ARGS` should be applied to all restic commands, and `RESTIC_BACKUP_EXTRA_ARGS` only to the backup command, we should keep `"${extra_args[@]}" ` here

**Review by erikw — COMMENTED** [review]
If I get the intention right, `RESTIC_EXTRA_ARGS` would be used to set "Global Flags" as described at

https://restic.readthedocs.io/en/latest/manual_rest.html

Is this correct?

I think we should add commented-out example to 
https://github.com/erikw/restic-automatic-backup-scheduler/blob/main/etc/restic/default.env.sh
just as there is for `RESTIC_BACKUP_EXTRA_ARGS` already.

**Comment by nettnikl** [comment]
Hey @erikw , sorry for the delay, i had a stressful time at work. Mind to take a look at the updates?

**Comment by erikw** [comment]
No worries, I understand :).

I fixed indentation and Changelog just in the PR. Thanks!

**Pull request merged**